token-client/src/main/java/com/sap/cloud/security/xsuaa/client/AbstractOAuth2TokenService.java

```diff
@@ -137,4 +168,78 @@ public OAuth2TokenResponse retrieveAccessTokenViaJwtBearerTokenGrant(URI tokenEn
 	protected abstract OAuth2TokenResponse requestAccessToken(URI tokenEndpointUri, HttpHeaders headers,
 			Map<String, String> parameters) throws OAuth2ServiceException;
 
+	private OAuth2TokenResponse getOAuth2TokenResponse(@Nonnull URI tokenEndpointUri, HttpHeaders headers,
+			Map<String, String> additionalParameters,
```

Comment:
pls rename additionalParameters to parameters

Reply:
done!